### ISSUE TITLE
Backup / Import data to new phone

### ISSUE THREAD

**Issue by Dzjed** [albertvaka/bettercounter#20]
I'm trying to backup my BetterCounter data because I've got a new phone. I can't access the files manually using ``adb shell`` (I assume they are in ``/data/user/0/org.kde.bettercounter``) because of ``Permission denied``. Using ``adb backup org.kde.bettercounter -f backup.ab`` I only get a useless 549 B file with only some meta data / encryption header. In the logs it says

```
Package org.kde.bettercounter is not eligible for backup, removing.
```

I can export the data, but since there is no way to import it again, how could this be solved? I do not want to loose over a year worth of data. Thanks.

**Comment by albertvaka** [comment]
If your phone is rooted you should be able to use tools like Titanium Backup.

If your phone is not rooted, you can enable Google Backups and it should automatically install the app and restore the data when you first sign in on the new phone.

**Comment by Dzjed** [comment]
My phone is not rooted and I'm on GrapheneOS without a Google account / services, so that unfortunately is not an option.

**Comment by albertvaka** [comment]
Then I'm not sure there's currently a way... Other than adding an import feature to the app. Let me see if I can  implement something.

**Comment by Dzjed** [comment]
That would be very useful in general in my opinion. I hope you can make this work!

**Comment by albertvaka** [comment]
I just added a very simple import feature in commit f19654ac59911cff4252c60ba273a17efe3aad0c.

I've published a new version of the app and it should soon (tomorrow?) be available both on F-Droid and Google Play.

Let me know how it works :)

Note that the imported counters will not preserve the interval (daily/monthly/etc) nor the color, since that info doesn't get exported.

Also, importing *doesn't clear the existing data*. So, if you already have some counts on the new phone they will just be added together with the imported ones.
